package/gluon-ebtables-net-rules/files/lib/gluon/ebtables/110-net-allow-ipv6

```diff
@@ -0,0 +1,9 @@
+siteConfig = require("gluon.site_config")
+
+rule ('NET_SRC -p IPv6 --ip6-src fe80::/10 -j RETURN')
+rule ('NET_SRC -p IPv6 --ip6-dst ff00::/8 -j RETURN')
```

Comment:
Ah, reports should be excluded for "::", too, they will happen just before the IPv6 DAD Neighbor Solicitations. So icmpv6 types 131 and 143.

(See https://tools.ietf.org/html/rfc3810#section-5.2.13., for instance)